admin/src/scenes/dashboardV2/components/General.jsx

```diff
@@ -44,17 +47,24 @@ export default function Index() {
         return ["general", "engagement", "sejour", "inscription"];
     }
   })();
+
+  const needMoreInfo = [ROLES.ADMIN, ROLES.REFERENT_REGION, ROLES.REFERENT_DEPARTMENT].includes(user.role);
+
   return (
-    <DashboardContainer active="general" availableTab={availableTab}>
-      <div className="flex flex-col gap-8 mb-4">
-        {message?.length ? message?.map((hit) => <InfoMessage key={hit._id} data={hit} />) : null}
-        <h1 className="text-[28px] font-bold leading-8 text-gray-900">En ce moment</h1>
-        <div className="flex w-full gap-4">
-          <Todos user={user} />
-          {user.role !== ROLES.HEAD_CENTER && <KeyNumbers role={user.role} />}
+    <Page>
+      {needMoreInfo && <BandeauInfo />}
+      <Header title="Tableau de bord" breadcrumb={[{ title: <HiOutlineChartSquareBar size={20} /> }, { title: "Tableau de bord" }]} />
+      <DashboardContainer active="general" availableTab={availableTab}>
+        <div className="flex flex-col gap-8 mb-4">
+          {message?.length ? message?.map((hit) => <InfoMessage key={hit._id} data={hit} />) : null}
```

Comment:
```suggestion
          {message?.map((hit) => <InfoMessage key={hit._id} data={hit} />)}
```
ça devrait suffire, si tu retourne `null` pas la peine d'alourdir la syntaxe